packages/icon/test/icon.test.js

```diff
@@ -161,6 +166,45 @@ describe('vaadin-icon', () => {
       icon.__fetch.restore();
     });
 
+    it('should append value from symbol property to src', () => {
+      icon.src = './icon.svg';
+      icon.symbol = 'symbol-id';
+
+      expect(svgElement.querySelector(`use[href="${icon.src}#${icon.symbol}"]`)).to.exist;
+    });
+
+    it('should use value from symbol when src path has a hash value', () => {
+      icon.src = './icon.svg#id-0';
+      icon.symbol = 'id-1';
+
+      expect(svgElement.querySelector(`use[href="${icon.src.split('#')[0]}#${icon.symbol}"]`));
```

Comment:
Oh, this expect doesn't seem complete. Should it have `to.exist` at the end?

Reply:
Nice catch. Added.